kubernetes-deployment/template/kubelet.sh.template

```diff
@@ -36,7 +36,8 @@ docker run \
     {{ clusterconfig['dockerregistry'] }}/hyperkube:{{ clusterconfig['hyperkubeversion'] }} \
     /hyperkube kubelet --address={{ hostcofig['hostip'] }} \
   --register-node=true \
-  --api-servers={{ clusterconfig['api-servers-ip'] }}:8080 \
+  --kubeconfig=/etc/kubernetes/config \
```

Comment:
>kubeconfig=/etc/kubernetes/config \ [](start = 4, length = 35)

if we change to a kubeconfig, should we include an example kube config ?

Reply:
K8s1.9 has removed the option api-servers. Instead, we should write the ip-address in the kubeconfig file.

Reply:
The same configuration of kubectl

Reply:
do we have an example kubeconfig for kubectl?

---
In reply to: [175327008](https://github.com/Microsoft/pai/pull/394#discussion_r175327008) [](ancestors = 175327008)

Reply:
Yes